pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -343,7 +346,6 @@ func (ns *nodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	volumeMutex.LockKey(req.GetVolumeId())
 	defer volumeMutex.UnlockKey(req.GetVolumeId())
 
-	// showing for debug:
 	klog.V(4).Infof("NodeUnStageVolume: VolumeID:%v Staging target path:%v",
```

Comment:
In this case, we can remove this logging?

Reply:
My idea was to have log msg in all handling functions near start, so that we see the start of handling. There is log message later, but in the middle there may be return with another msg. So my reasoning has been, lets start each handler with message. Level is 4 so we see this only in debug and test cases. I wanted to remove comment above log line because this does not add more value, level 4 of logging shows already it's for dev. purposes.

Reply:
> My idea was to have log msg in all handling functions near start, so that we see the start of handling.

I fully agree with your point. My point was we can remove the existing log message in line 349, which is a duplicate of this newly added logging line. Or I missing something?

Reply:
Opps sorry, both are diffrent functions. Nevermind